### PR TITLE
Update snapshots.md

### PR DESCRIPTION
**Pull request by runleonarun**
## Description & motivation

Per @mirnawong1's [feedback](https://dbt-labs.slack.com/archives/C02NCQ9483C/p1649065071386239), the two details blocks are not showing up in the bottom of the page. I think they might just need to include a summary?

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [ ] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

### CODE REVIEW

File: website/docs/docs/building-a-dbt-project/snapshots.md [hunk 1]
```diff
@@ -377,7 +377,7 @@ For the `check` strategy, the current timestamp is used to populate each column
 
 <details>
 
-Snapshot query results at `2019-01-01 11:00`:
+    <summary>Snapshot query results at `2019-01-01 11:00`</summary>
```
Comment (runleonarun) [inline website/docs/docs/building-a-dbt-project/snapshots.md]:
```suggestion
<summary>Snapshot query results at `2019-01-01 11:00`</summary>
```

**Comment by runleonarun** [comment]
When I come back to this:
* Move content out of details tabs
* Can we reconfigure this content to make it easier to read?
* Add context requested from @pofl 
 > I do want to make a small request here. The docs do not mention what type the updated_at field of the timestamp strategy should have. timestamp types work, of course. But what about ISO date strings and unix epoch integers? Please add this (one sentence) to the docs. You can even hijack this PR to add that.

**Comment by nghi-ly** [comment]
@runleonarun : So sorry for the delay in getting this reviewed! As you might have heard, the Docs team will be dedicating time on a regular cadence to manage our PR backlog better. In the spirit of this, I opened this PR to see what I can do to help it along.

Your changes look good and ready for approval but I wasn't sure whether you were going to work on this PR further. Are you ready for me to review this or do you need to make more changes? Thanks!

**Comment by runleonarun** [comment]
Thanks for the nudge @nghi-ly! I moved the additional thoughts for improvement over to an [issue](https://github.com/dbt-labs/docs.getdbt.com/issues/1948) so we can get this merged and fix the error. Can you please review it as is? Thank you! 🙇🏻‍♀️

**Review by nghi-ly — APPROVED** [review]

**Pull request merged**